app/league/dashboard/views.py

```diff
@@ -173,7 +173,7 @@ def create_game():
             episode=form.episode.data,
             played_at=played_at
         )
-        messenger.notify_slack({'text':repr(game)})        
+        messenger.notify_slack(repr(game))
```

Comment:
Well, that's something that _you'd_ want.

Is it a bad idea to make this modification in the game class?